e2e/tests/0050-greenboot.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+PS4='+ $(date "+%T.%N")\011 '
+set -x
+
+SCRIPT_PATH="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")"
+
+scp "${SCRIPT_PATH}/../../docs/config/busybox_running_check.sh" "${SCRIPT_PATH}/assets/greenboot-test.sh" "${USHIFT_USER}@${USHIFT_IP}":/tmp/
```

Comment:
Let's copy those to the user home dir if possible to avoid security remarks about predictable file names in /tmp?